src/FilesAndStreams/ParallelFileProcessing.cs

```diff
@@ -0,0 +1,35 @@
+﻿// <copyright file="ParallelFileProcessing.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace FilesAndStreams
+{
+    /// <summary>
+    /// ParallelFileProcessing - Parallel creates three files and writes in three files.
+    /// </summary>
+    public class ParallelFileProcessing
+    {
+        /// <summary>
+        /// Gets or Sets filepaths.
+        /// </summary>
+        /// <value>FilePaths - List of file paths</value>
+        public List<string> FilePaths { get; set; } = new () { "file1.txt", "file2.txt", "file3.txt" };
+
+        /// <summary>
+        /// ProcessFileConcurrently - Takes the path of the file from the list and writes concurrently.
+        /// </summary>
+        public void ProcessFileConcurrently()
+        {
+            int count = 0;
+            Parallel.ForEach(this.FilePaths, (file) =>
+            {
+                StreamWriter streamWriter = new (file);
+                streamWriter.WriteLine($"Processed by thread : {Thread.CurrentThread.ManagedThreadId}");
```

Comment:
Also add console writeline for this as well
```suggestion
                streamWriter.WriteLine($"Processed by thread : {Thread.CurrentThread.ManagedThreadId}");
                Console.WriteLine($"Processed by thread : {Thread.CurrentThread.ManagedThreadId}");
```

Reply:
WriteLine() included.